src/Interactive/Features/Interactive/Core/InteractiveHost.RemoteService.cs

```diff
@@ -116,34 +126,50 @@ internal void Dispose(bool joinThreads)
                 // set _disposing so that we don't attempt restart the host anymore:
                 _disposing = true;
```

Comment:
>_disposing = true; [](start = 16, length = 18)

This should also be inside the using.

Reply:
Actually we don't need _disposing variable anymore.

---
In reply to: [162233741](https://github.com/dotnet/roslyn/pull/24058#discussion_r162233741) [](ancestors = 162233741)

Reply:
Agree. For consistency, it is better to put it inside using.

---
In reply to: [162233741](https://github.com/dotnet/roslyn/pull/24058#discussion_r162233741) [](ancestors = 162233741)

Reply:
Not just for consistency. Since this is now shared variable it would need to be under the lock. But we don't need it at all actually.

---
In reply to: [162234150](https://github.com/dotnet/roslyn/pull/24058#discussion_r162234150) [](ancestors = 162234150,162233741)